apps/system/test/marionette/task_manager_icons_test.js

```diff
@@ -94,13 +101,16 @@ marionette('Task Manager - Icons', function() {
                icon.getAttribute('class').indexOf('pending') === -1;
       });
 
-      var expectedColor = '0,255,255,255';
-      var expectedSize = 32;
+      // The system will select an icon based on the current devicePixelRatio.
+      var expectedSize = 32 * client.executeScript(function() {
```

Comment:
Shouldn't the expected size be '16 || 32 || 64 || 128 || 256'? If I understand this correctly, that code won't work. We should find the closest size to 32 * devicePixelRatio.